### DISCUSSION TITLE
Purpose

### DISCUSSION THREAD

**Discussion by Tolriq** [opensubsonic/open-subsonic-api#1] [Announcements]
This repository is the result of a couple of months of discussions between some Subsonic servers and clients to try to improve the currently limited and abandoned Subsonic API.

This allows the Subsonic ecosystem to improve, without trying to rebuild a complete new API.

To simplify servers adoption and client life, the current objective of the version 1 of Open Subsonic API is to be build as extensions over the current subsonic API. With most of the features being optional but properly documented.

There's still a lot of things to define and build, but the concept and ideas have already matured enough to open this project to more people to build the proper extensions.

Most of the previous discussions will be migrated to this repository but currently the history can be seen at:
- https://support.symfonium.app/t/expanding-subsonic-api/1115
- https://support.symfonium.app/c/subsonic-api-extensions/11

**Comment by deluan** [comment]
Hey all @opensubsonic/clients / @opensubsonic/servers 

Let's show our support to the project, by making our participation in the organization public. Just go to https://github.com/orgs/opensubsonic/people and change your visibility to "Public":
<img width="1077" alt="Screenshot 2023-04-07 at 1 13 49 PM" src="https://user-images.githubusercontent.com/331353/230649749-3764910e-8060-4d05-9d13-fa5aadf47b13.png">
